### PR TITLE
Build and implement side navigation

### PR DESCRIPTION
**Pull request by Susanne1307**
Created a side navigation that can be opened and closed:

Screen-Recording-2020-04-19-at-1

Preview: 
https://foosbuddy-pi-menu-o8c8vp4iufiv.herokuapp.com/search

### CODE REVIEW

**Comment by florianGierlichs** [comment]
Good job, nice you implement react context!
The options "Overview", "Add searcht" ect.  in your sidebar look like input fields with placeholders in my opinion. I was a bit confused when I saw them the first time and realized, they are anchor elements.
This is only an optical  thing, so nothing to worry about too much, but I thought, I share it anyway 😄

File: package.json [hunk 1]
```diff
@@ -51,7 +51,8 @@
   "dependencies": {
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mongodb": "^3.5.5"
+    "mongodb": "^3.5.5",
+    "prop-types": "^15.7.2"
```
Comment (marwinburesch) [inline package.json:55]: doesn't belong in this PR
